test/acceptance/behave/components/ngsiv2/attributes_values/get_attribute_value/get_attribute_value.feature

```diff
@@ -753,7 +752,7 @@ Feature: get an attribute value by entity ID and attribute name if it exists usi
       | error       | BadRequest        |
       | description | service not found |
 
-  @entity_id_wrong
+  @entity_id_wrong  @BUG_1887 @skip
```

Comment:
yes, it is scenario and another with invalid attr_name using house_# return an 406 http code. Commented in the issue #1887